packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -246,7 +254,8 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
     componentDidMount(): void {
         const { selectStep, tab } = this.props;
 
-        if (tab !== EntityInsertPanelTabs.First) {
+        const tabs = this.getTabs();
+        if (tab !== EntityInsertPanelTabs.First && tabs.length > EntityInsertPanelTabs.First) {
```

Comment:
The comparison of `length` to the `First` enum value seems odd semantically. Could it just use the literal 1 instead?